target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -71,6 +71,11 @@ comfast,cf-e5)
 	ucidef_set_led_rssi "rssimedium" "RSSIMEDIUM" "$boardname:blue:rssi1" "wlan0" "33" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:blue:rssi2" "wlan0" "66" "100"
 	;;
+devolo,dlan2-2400-ac|\
+devolo,magic2wifi)
+	ucidef_set_led_netdev "plcw" "dLAN" "devolo:status:dlan" "eth0.1" "rx"
+	ucidef_set_led_wlan "wlan" "WLAN" "devolo:status:wlan" "phy1tpt"
```

Comment:
This can be moved to the DTS.

Reply:
Is there a way to move this to DTS?
     ucidef_set_led_netdev "plcw" "dLAN" "devolo:status:dlan" "eth0.1" "rx"
This is what I have on the DTS: 
     linux,default-trigger = "netdev";
I don´t know how to specify eth0.1 and rx... (or ideally rx and tx)

Reply:
I was talking about the phy1tpt, which you already have in DTS, so you can just remove that line.
For the netdev, I haven't seen a DTS version so far.